src/sequelize/models/User.js

```diff
@@ -62,8 +62,10 @@ export default (sequelize, DataTypes) => {
     }
   );
 
-  User.associate = () => {
+  User.associate = (models) => {
     // Association definitions here
+    User.hasMany(models.Follow, { foreignKey: 'following' });
```

Comment:
you should add `onDelete: 'CASCADE'` here so that the db is not in an inconsistent state when a user is deleted. The next line as well

Reply:
Yes, It is added.